### DISCUSSION TITLE
Description above index cards

### DISCUSSION THREAD

**Discussion by jcubic** [facebook/docusaurus#10767] [Q&A] [answered]
Is there a simple way to add description above the cards in documentation index page?

![Przechwycenie obrazu ekranu_2024-12-14_16-40-09](https://github.com/user-attachments/assets/d5c0c053-1a57-4fa4-a671-4b1ed3fe1067)

I want to add a bit of information about this part of the documentation.

**Comment by jcubic** [comment, marked as answer]
I've setup a new project and found that there is a description in `_category_.json` file.

**Reply by slorber** [reply]
For more advanced cases you can also create a `category/index.mdx` and render the `<DocCardList>` component anywhere you want.

Doc: https://docusaurus.io/docs/sidebar/items#embedding-generated-index-in-doc-page

Examples:
- https://docusaurus.io/docs/sidebar
- https://docusaurus.io/docs/migration

**Reply by jcubic** [reply]
Thanks it seems that this is what I need, I've tried to put Markdown code into description, but it didn't work. This should solve the issue.

I've also noticed that JSX works in both mdx and md files. So it looks like mdx extension is like documentation that it contain components inside.

**Reply by slorber** [reply]
Because both extensions are currently parsed the same way as MDX for historical reasons.


But you can opt-in for CommonMark mode and use HTML tags instead of JSX, see https://docusaurus.io/docs/markdown-features#mdx-vs-commonmark

In the future `.md` files will be parsed as CommonMark by default. Still, we could implement the same functionality thanks to a markdown directive such as `::theme-DocCardList` so that you can achieve the same with both CommonMark or MDX.
